lenskit-core/src/main/java/org/grouplens/lenskit/vectors/similarity/PearsonCorrelation.java

```diff
@@ -57,6 +58,7 @@ public PearsonCorrelation() {
 
     @Inject
     public PearsonCorrelation(@SimilarityDamping double s) {
+        Preconditions.checkArgument(s >= 0, "negative damping not allowed");
```

Comment:
There was one more place I found in a usage search of `@SimilarityDamping`. Fixed in a59a93e.